.circleci/config.yml

```diff
@@ -412,7 +412,7 @@ workflows:
               only: /(^build$)|(^[v]?[0-9]+(\.[0-9]+)*$)/
             branches:
               # Always build on master
-              only: /^.*(master|build).*$/
+              only: /^.*(master|[0-9]+.[0-9]+.[0-9]+).*$/
```

Comment:
Now that I am looking into it, I wonder why we decided to add the leading and trailing `.*`. Maybe @pawelchcki or @SammyK have a better memory. Unless any specific reason let's remove it.

Reply:
Thanks for spotting this @labbati , I'm using `build` suffix to a branch name to trigger all the package builds whenever I'm testing changes that could affect that step.

If possible I'd like to retain that ability, or have some other way to trigger those builds.

Reply:
Ah sure I think that this was only there from back in the days when we wanted to trigger builds any time. Then @morrisonlevi in the end the final version will be something like

```
only: /^.*(master|build|[0-9]+.[0-9]+.[0-9]+).*$/
```